### ISSUE TITLE
How to acess point cloud normals?

### ISSUE THREAD

**Issue by SubChange** [CloudCompare/CloudComPy#134]
I note that there are methods to get point coordinates and colors of a point cloud (`toNpArrayCopy` and `colorsToNpArray`), are there any methods to get normals of the point cloud after I compute them using `cloudComPy.PCL.NormalEstimation`?

**Comment by prascle** [comment]
You are right, there is no easy way to get the normals as an array. I will add that to my TODO list, for the next release.
As a workaround, it's possible to export the normal as scalar fields and then to get arrays associated to the scalar fields.
Regards,
Paul

**Comment by SubChange** [comment]
> You are right, there is no easy way to get the normals as an array. I will add that to my TODO list, for the next release. As a workaround, it's possible to export the normal as scalar fields and then to get arrays associated to the scalar fields. Regards, Paul

That's great, thank you for your excellent work.

**Comment by PMDevine1** [comment]
In addition, it would also be useful to have a function that sets the pc normals from a nparray similar to the coordsFromNPArray_copy function.

**Comment by prascle** [comment]
OK, that should be doable!
Best,
Paul
